install_config/persistent_storage/topics/glusterfs_using_storage.adoc

```diff
@@ -43,15 +43,15 @@ it does not already exist:
 ----
 # oc get pods -o wide
 NAME                               READY     STATUS    RESTARTS   AGE       IP               NODE
-nginx-pod                          1/1       Running   0          9m        10.38.0.0        node1
+hello-openshift-pod                          1/1       Running   0          9m        10.38.0.0        node1
 ----
 
 . `oc exec` into the container and create an *_index.html_* file in the
 `mountPath` definition of the pod:
 +
 ----
-$ oc exec -ti nginx-pod /bin/sh
-$ cd /usr/share/nginx/html
+$ oc exec -ti hello-openshift-pod /bin/sh
+$ cd /usr/share/hello-openshift/html
```

Comment:
The path coming from https://github.com/openshift/openshift-docs/pull/10618/files#diff-492418d8b4b3aa3ae12668611e127d51R24 should not be updated.